pipeline_dp/mean_var_params.py

```diff
@@ -0,0 +1,123 @@
+"""DP computing of count, sum, mean, variance."""
+
+import numpy as np
+import pipeline_dp
+
+from dataclasses import dataclass
+
+
+@dataclass
+class MeanVarParams:
+	"""Manages the parameters used for computing the mean/variance."""
+	eps: float
+	delta: float
+	low: float
+	high: float
+	max_partitions_contributed: int
+	max_contributions_per_partition: int
+	noise_kind: pipeline_dp.NoiseKind  # Laplace or Gaussian
+
+
+def _apply_laplace_mechanism(value, dp_params):
+	"""Applies the Laplace mechanism.
+
+	Args:
+		value: The result of querying the database.
+		dp_params: The parameters used at computing the noise.
+	"""
+	l1_sensitivity = dp_params.max_partitions_contributed * dp_params.max_contributions_per_partition
```

Comment:
Sensitivity depends on the type of aggregation. Namely sensitivity is a maximum change by adding/removing one user. This formula works when each contribution is 1 (like in count). For sum, each contribution is within [low, high], so the maximum contribution is max(|low|, |high|). 

Please please add methods for computation l1_sensitivity/l2_sensitiviy to MeanVarParams  (like count_l1_sensitivity, sum_l1_sensitivity, count_l2_sensitivity, sum_l2_sensitivity)

Reply:
Thanks for the explanation! Does the aggregation type only affect linf_sensitivity? If so, l1_sensitivity and l2_sensitivity have the same dependency on l0_sensitivity and linf_sensitivity. I created functions for computing l0_sensitivity/linf_sensitivity based on the type of aggregation and two general functions that compute l1_sensitivity/linf_sensitivity given l0_sensitivity & linf_sensitivity.